backend/pkg/web/handler/auth.go

```diff
@@ -2,33 +2,73 @@ package handler
 
 import (
 	"fmt"
+	"net/http"
+
 	"github.com/fastenhealth/fasten-onprem/backend/pkg"
 	"github.com/fastenhealth/fasten-onprem/backend/pkg/auth"
 	"github.com/fastenhealth/fasten-onprem/backend/pkg/config"
 	"github.com/fastenhealth/fasten-onprem/backend/pkg/database"
 	"github.com/fastenhealth/fasten-onprem/backend/pkg/models"
 	"github.com/fastenhealth/fasten-onprem/backend/pkg/utils"
 	"github.com/gin-gonic/gin"
-	"net/http"
+	"github.com/sirupsen/logrus"
 )
 
+type UserWizard struct {
+	*models.User    `json:",inline"`
+	JoinMailingList bool `json:"join_mailing_list"`
+}
+
+func RequireAdmin(c *gin.Context) {
```

Comment:
As mentioned elsewhere, we might want to replace this with a `IsAdmin` function, or return an error value so we can short-circuit all subsequent processing.